src/contributing.md

```diff
@@ -373,9 +373,12 @@ the submodule to. Running `./x.py build` should work now.
 ## Writing Documentation
 
 Documentation improvements are very welcome. The source of `doc.rust-lang.org`
-is located in `src/doc` in the tree, and standard API documentation is generated
-from the source code itself. Documentation pull requests function in the same way
-as other pull requests.
+is located in [`src/doc`][srcdoc] in the tree, and standard API documentation is generated
+from the source code itself (e.g. [`lib.rs`][librs]). Documentation pull requests function
```

Comment:
You could shorten the lines by using implicit reference links:
```suggestion
is located in [`src/doc`] in the tree, and standard API documentation is generated
from the source code itself (e.g. [`lib.rs`]). Documentation pull requests function
```
and then
```
[`src/doc`]: ...
```
below

Reply:
Thanks for the suggestion!

Now that you mention it, I do recall seeing that technique used somewhere. :)

Shall I "commit suggestion" on your version or not worry now that the lines are under the limit anyway?

Reply:
Personally I like the shorter version, but it's up to you.

Reply:
Note that 'commit suggestion' is not sufficient because you have to update the reference links.

Reply:
Have changed to implicit reference links so will mark this as resolved.